dcat/rdf/dcat.ttl

```diff
@@ -1594,30 +1597,31 @@ prov:wasGeneratedBy
   skos:scopeNote "foaf:homepage es la propiedad funcional inversa (IFP por sus siglas en inglés), lo que significa que DEBE ser única y identificar la página web del recurso percisamente. Esta propiedad indica la página web canónica, que pude ser útil en casos en los que hay más de una página web para el recurso."@es ;
   skos:scopeNote "foaf:homepage je inverzní funkčí vlastnost, což znamená, že MUSÍ být unikátní a MUSÍ přesně označovat webovou stránku zdroje. Označuje hlavní webovou stránku, což se může hodit v případě, že webových stránek o zdroji existuje více."@cs ;
 .
 time:hasBeginning
-  skos:scopeNote "Use of the property time:hasBeginning entails that value of the dct:temporal property is a member of the time:TemporalEntity class from [OWL-TIME]. In this context this could be taken to imply that dct:PeriodOfTime is equivalent to the subclass time:ProperInterval"@en ;
+  skos:scopeNote "Use of the property time:hasBeginning entails that value of the dct:temporal property is a member of the time:TemporalEntity class from [OWL-TIME]. In this context this could be taken to imply that dct:PeriodOfTime is equivalent to the subclass time:ProperInterval" ;
   skos:scopeNote "El uso de la propiedad time:hasBeginning implica que el valor de la propiedad dct:temporal es un miembro de la clase time:TemporalEntity de [OWL-TIME]. En este contexto, esto puede interpretarse como que implica que dct:PeriodOfTime es equivalente a la subclase time:ProperInterval"@es ;
   skos:scopeNote "Použití vlastnosti time:hasBeginning znamená, že hodnota vlastnosti dct:temporal je instancí třídy time:TemporalEntity ze slovníku [OWL-TIME]. V tomto kontextu to může znamenat, že dct:PeriodOfTime je ekvivalentní podtřídě time:ProperInterval"@cs ;
-  skos:changeNote "Property added in this context in this revision of DCAT"@en ;
-  skos:changeNote "Esta propiedad se agregó en este contexto en esta revisión de DCAT"@es ;
-  skos:changeNote "Vlastnost v tomto kontextu byla přidána v této revizi DCAT"@cs ;
+  skos:changeNote "Property added in this context in DCAT 2.0."@en ;
+  skos:changeNote "Esta propiedad agregado en DCAT 2.0."@es ;
+  skos:changeNote "Vlastnost v tomto kontextu byla přidána v revizi DCAT 2.0"@cs ;
 .
 time:hasEnd
-  skos:scopeNote "Use of the property time:hasEnd entails that value of the dct:temporal property is a member of the time:TemporalEntity class from [OWL-TIME]. In this context, this could be taken to imply that dct:PeriodOfTime is equivalent to the subclass time:ProperInterval"@en ;
+  skos:scopeNote "Use of the property time:hasEnd entails that value of the dct:temporal property is a member of the time:TemporalEntity class from [OWL-TIME]. In this context this could be taken to imply that dct:PeriodOfTime is equivalent to the subclass time:ProperInterval" ;
```

Comment:
```suggestion
  skos:scopeNote "Use of the property time:hasEnd entails that value of the dct:temporal property is a member of the time:TemporalEntity class from [OWL-TIME]. In this context this could be taken to imply that dct:PeriodOfTime is equivalent to the subclass time:ProperInterval"@en ;
```